mailpoet/assets/js/src/automation/sections/templates-section.tsx

```diff
@@ -0,0 +1,33 @@
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { MailPoet } from '../../mailpoet';
+import { workflowTemplates } from '../templates/config';
+import { TemplateListItem } from '../templates/components/template-list-item';
+
+export function TemplatesSection(): JSX.Element {
+  const templates = workflowTemplates.slice(0, 3);
+
+  return (
+    <section className="mailpoet-automation-section mailpoet-section-templates">
+      <h2>{__('Start with essentials', 'mailpoet')}</h2>
+      <p>
+        {__(
+          'Choose from our list of pre-made templates and make it your own',
+          'mailpoet',
+        )}
+      </p>
+      <ul className="mailpoet-section-template-list">
+        {templates.map((template) => (
+          <TemplateListItem
```

Comment:
For some reason those don't seem to have the same hover effect as templates do.

Reply:
dfee86273

Reply:
After rebase: [0cbddaf](https://github.com/mailpoet/mailpoet/pull/4461/commits/0cbddafb899ce53abe1f4ba515ce920b6167352e)